Xamarin.Forms.Platform.UAP/Properties/Xamarin.Forms.Platform.UAP.rd.xml

```diff
@@ -28,6 +28,7 @@
   <Library Name="Xamarin.Forms.Platform.UAP">
 
   	<!-- add directives for your library here -->
-
+    <Namespace Name="Windows.UI.Xaml" MarshalObject="Required All"/>
```

Comment:
We can actually just do `<Namespace Name="Windows.UI.Xaml.Automation.Peers" MarshalObject="Required All"/> `. I think bringing in the whole namespace is bloaty.